etc/xivo/asterisk/xivo_in_callerid.conf

```diff
@@ -27,6 +27,12 @@ callerid = ^\+33[1-9]\d{8}$
 strip = 3
 add = 0
 
+[national4]
+comment = local number with international prefix, without "+" followed by 9 digits
+callerid = ^33[1-9]\d{9}$
```

Comment:
I think there's an error in the ticket reporting this issue.

Based on the `national3` rule I think it should be `^33[1-9]\d{8}$`

A number in Île de France would be `+33 1 xx xx xx xx` in international format. The national format should be `01 xx xx xx xx`

I'd like you to double check with some knowledgeable with french numbers.

Reply:
alright, thanks!

Reply:
That's right: national numbers are 10 digits long, international numbers are prefixed with `+33` then 9 digits